silkrpc/types/transaction.hpp

```diff
@@ -37,6 +37,10 @@ struct Transaction : public silkworm::Transaction {
     intx::uint256 effective_gas_price() const; // EIP-1559
 };
 
+struct Rlp  {
+   silkworm::Bytes rlp;
```

Comment:
Probable `buffer` is better than `rlp` as member variable name here to avoid `rlp.rlp` at call site.